documentation/source/conf.py

```diff
@@ -32,74 +38,44 @@
 extensions = ["breathe", "sphinx.ext.mathjax", "sphinx.ext.ifconfig"]
 
 read_the_docs_build = os.environ.get("READTHEDOCS", None) == "True"
-travis_build = os.environ.get("TRAVIS_CI", None) == "True"
-
-# Get a description of the current position. Use Popen for 2.6 compat
-git_tag = subprocess.Popen(["git", "describe", "--tags"], stdout=subprocess.PIPE).communicate()[0]
 
-# convert from bytes to string
-git_tag = git_tag.decode("ascii")
-
-if travis_build:
-
-    # Don't attempt to set the path as breathe is installed to virtualenv on travis
-
-    # Set values with simple strings
-    version = "'travis'"
-    release = "'travis'"
-    documentation_build = "travis"
-
-elif read_the_docs_build:
+# The version info for the project you're documenting, acts as replacement for
+# |version| and |release|, also used in various other places throughout the
+# built documents.
+#
+# The short X.Y version.
+version = "'unknown'"
+# The full version, including alpha/beta/rc tags.
+release = "'unknown'"
 
-    # On RTD we'll be in the 'source' directory
-    sys.path.append("../../")
+# Get a description of the current position. Use Popen for 2.6 compat
```

Comment:
The `Popen` note no longer applies to the new code right? Unless I'm seeing something wrong.

Reply:
Yes, if py2.6 is not supported. I think I read somewhere in this repo's history that py2.x support was dropped.